packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-file/gux-form-field-file.tsx

```diff
@@ -136,25 +115,12 @@ export class GuxFormFieldFile {
   render(): JSX.Element {
     return (
       <GuxFormFieldFieldsetContainer labelPosition={this.computedLabelPosition}>
```

Comment:
If the legend is gone/no needed can the fieldset go too?
```
      <GuxFormFieldContainer labelPosition={this.computedLabelPosition}>
````

Reply:
Good catch